system-x/services/splunk/src/main/java/software/tnb/splunk/service/openshift/Splunk.java

```diff
@@ -0,0 +1,220 @@
+package software.tnb.splunk.service.openshift;
+
+import software.tnb.common.account.AccountFactory;
+import software.tnb.common.config.OpenshiftConfiguration;
+import software.tnb.common.deployment.OpenshiftDeployable;
+import software.tnb.common.deployment.WithExternalHostname;
+import software.tnb.common.openshift.OpenshiftClient;
+import software.tnb.common.service.Service;
+import software.tnb.common.utils.WaitUtils;
+import software.tnb.splunk.account.SplunkAccount;
+import software.tnb.splunk.validation.SplunkValidation;
+
+import org.apache.commons.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.auto.service.AutoService;
+import com.google.common.io.Resources;
+import com.splunk.ServiceArgs;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import cz.xtf.core.openshift.helpers.ResourceParsers;
+import io.fabric8.kubernetes.api.model.DeletionPropagation;
+import io.fabric8.kubernetes.api.model.HasMetadata;
+import io.fabric8.kubernetes.api.model.IntOrString;
+import io.fabric8.kubernetes.api.model.Pod;
+import io.fabric8.kubernetes.api.model.apiextensions.v1.CustomResourceDefinition;
+import io.fabric8.kubernetes.api.model.rbac.ClusterRole;
+import io.fabric8.kubernetes.client.dsl.base.CustomResourceDefinitionContext;
+import io.fabric8.openshift.api.model.Route;
+import io.fabric8.openshift.api.model.RouteBuilder;
+
+@AutoService(Splunk.class)
+public class Splunk implements Service, OpenshiftDeployable, WithExternalHostname {
```

Comment:
this should be divided into LocalSplunk and OpenshiftSplunk

Reply:
if you don't want to implement the local variant, just create empty classes and please create fuseqe jira with component tnb for that

Reply:
also consired of the openshift variant could implement `ReusableOpenshiftDeployable` - when multiple tests are using it, it is not recreated everytime, but it keeps running instead and it is cleaned via the `cleanup` method (possibly delete all indices there?)

Reply:
I thought that only installation via operator is possible but I have already found a docker image with an example for Splunk "standalone". So I am going to create also local variant.

Reply:
@avano There are also system indexes so one way is to store before starting (in `openResources` method) what indexes should be ignored for the deletion or try to delete it and ignore the exception ( that index is internal or disabled ). 
Or ignore `cleanup` method at all